packages/cli/src/commands/dbCommands/generate.js

```diff
@@ -21,6 +21,12 @@ export const builder = (yargs) => {
       description: 'Print more',
       type: 'boolean',
     })
+    .option('schema', {
```

Comment:
Since this option is shared with almost all the commands, maybe we can export the object somewhere and re-use it in all the other commands? DRY 'n all that.

I would stick it in `commands/dbCommands/options.ts`:
`export const schema = () => ({ ... })`

And then when you use it:

`.option('schema', schema())`